lib/vsc/utils/generaloption.py

```diff
@@ -752,6 +751,10 @@ def get_env_options_prefix(self):
 
     def get_env_options(self):
         """Retrieve options from the environment: prefix_longopt.upper()"""
+        if not self.environment_arguments is None:
+            # already done, let's not do this again
+            return self.environment_arguments
```

Comment:
this is required in order not to break the `test_error_env_options` options, since otherwise `self.error_env_option_method` gets called twice, because `get_env_options()` is called twice via `_get_args()`

But, with this 'fix' in place, the `test_loglevel` test is broken. It seems to indicate that `--quiet` no longer gets preference over (the test equivalent of) `$EASYBUILD_INFO`.

Interesting...